ironfish/src/wallet/walletdb/transactionValue.ts

```diff
@@ -85,6 +112,8 @@ export class TransactionValueEncoding implements IDatabaseEncoding<TransactionVa
     if (value.sequence) {
       size += 4
     }
+    size += 4
+    size += value.assetBalanceDeltas.size * (ASSET_IDENTIFIER_LENGTH + 8)
```

Comment:
ASSET_IDENTIFIER_LENGTH is in bits but these are bytes right?

Reply:
I wasn't sure, but it looks like ASSET_IDENTIFIER_LENGTH is 32 bytes: https://github.com/iron-fish/ironfish/blob/38d2528903d3def6be0eaa4eaf400deb541b404c/ironfish-zkp/src/constants.rs#L12